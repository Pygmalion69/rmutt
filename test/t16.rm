// positional arguments
foo: "it's true, " bar("this","that");
bar(a,b): "i like " _1 " and " b;