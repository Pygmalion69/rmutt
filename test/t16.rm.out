it's true, i like this and that
